99-make_reports.py

```diff
@@ -118,10 +120,27 @@ def run_report(subject, session=None):
             if op.exists(fname_stc + "-lh.stc"):
                 stc = mne.read_source_estimate(fname_stc, subject)
                 _, peak_time = stc.get_peak()
-                brain = stc.plot(views=['lat'], hemi='both',
-                                 initial_time=peak_time)
-                fig = brain._figures[0]
-                rep.add_figs_to_section(fig, evoked.condition)
+
+                # Plot using 3d backend if available, and use Matplotlib
+                # otherwise.
+                if mne.viz.get_3d_backend() is not None:
+                    brain = stc.plot(views=['lat'], hemi='both',
+                                     initial_time=peak_time,  backend='mayavi')
+                    rep.add_figs_to_section(brain._figures[0],
+                                            evoked.comment)
+                else:
+                    import matplotlib.pyplot as plt
+                    fig_lh = plt.figure()
+                    fig_rh = plt.figure()
+
+                    brain_lh = stc.plot(views='lat', hemi='lh',
```

Comment:
btw these matplotlib plots look kinda awful (but at least better than nothing)
any ideas how to increase their resolution?